src/Event.php

```diff
@@ -674,6 +709,12 @@ public function toArray(): array
             $data['breadcrumbs']['values'] = $this->breadcrumbs;
         }
 
+        if (!empty($this->spans)) {
+            $data['spans'] = array_map(function ($span) {
```

Comment:
The `Closure` can be `static`, and it's missing both the typehint of the parameter and of the return value